package-lock.json.unit.test.mjs

```diff
@@ -55,7 +55,16 @@ describe('package-lock.json', () => {
       })
     })
 
-    it("should maintain `puppeteer` workspace 'devDependencies' compatibility", () => {
+    /**
+     * When this check fails, the incorrect `package-lock.json` can be repaired
+     * by checking out the Dependabot branch and updating `shared/helpers/package.json`
+     * so that the version of `puppeteer-core` in `peerDependencies` is compatible
+     * with that of `puppeteer` in the other workspaces (usually the version that Dependabot
+     * will have bumped `puppeteer` to).
+     *
+     * {@link https://team-playbook.design-system.service.gov.uk/how-we-work/version-control/pull-requests#reviewing-a-pr-from-dependabot}
```

Comment:
Broken link for me

Reply:
Oh, indeed, it has moved to https://team-playbook.design-system.service.gov.uk/guides/version-control/pull-requests/#reviewing-a-pr-from-dependabot. I'll update :)